test/e2e/mock/dl.js

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable camelcase */
+export const score_ad = (adMetadata, bid, auctionConfig, trustedScoringSignals, browserSignals) => {
```

Comment:
```suggestion
export const scoreAd = (adMetadata, bid, auctionConfig, trustedScoringSignals, browserSignals) => {
```

Reply:
current branch (and main still expect score_ad)